spec/unit/hanami/config/router_spec.rb

```diff
@@ -29,7 +29,7 @@
   context "hanami-router is not bundled" do
     before do
       allow(Hanami).to receive(:bundled?).and_call_original
-      expect(Hanami).to receive(:bundled?).with("hanami-router").and_return(false)
+      allow(Hanami).to receive(:bundled?).with("hanami-router").and_return(false)
```

Comment:
This change was needed because the `expect` led to the `false` only being returned for the first call, but now I've tidied up the initialization in `Config` (so that `@middleware` and `@router` are assigned separately), we needed this `bundled?` check to be called twice.